src/gnatss/loaders.py

```diff
@@ -142,11 +142,13 @@ def load_travel_times(
     if not is_j2k:
         from .utilities.time import AstroTime
 
-        # Determine j2000 time from date and time string
+        # Determine j2000 time from date and time string,
+        # assuming that they're in Terrestrial Time (TT) scale
```

Comment:
Is there anyway to assert/check that they are indeed in TT scale? what happens if they are not?

Reply:
> Is there anyway to assert/check that they are indeed in TT scale? what happens if they are not?

This is a great point. At this point, we know it's this since it's what got outputted from a proprietary software. Though in the future, probably good for user to set the scale to ensure it's valid... thinking about that, maybe this should be an argument.